src/apps/lwaftr/icmp.lua

```diff
@@ -93,6 +94,22 @@ function new_icmpv4_packet(from_eth, to_eth, from_ip, to_ip, initial_pkt, l2_siz
    return new_pkt
 end
 
+function is_icmpv4(pkt)
+   return rd16(pkt.data + constants.o_ethernet_ethertype) == constants.n_ethertype_ipv4
```

Comment:
I prefer leaving these local, and then moving them as soon as we actually concretely have a second caller in a different file actually re-using them. WDYT?